app/main.py

```diff
@@ -1,3 +1,78 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self: str) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Union[Distance, int, float]) -> Distance:
```

Reply:
use` from __future__ import annotations`

Reply:
It's  done)